server/server.js

```diff
@@ -0,0 +1,13 @@
+const http = require('http')
+const port = 3000
```

Comment:
Constant that almost never get changed are usually named in upper case (e.g. `PORT`).